src/components/info-tile/info-tile.types.ts

```diff
@@ -7,5 +7,5 @@ export interface InfoTileProgress {
     maxValue: number;
     prefix?: string;
     suffix: string;
-    displayPercentageColors?: boolean;
+    displayPercentageColors: boolean;
```

Comment:
Without the `?`, the consumer _has_ to provide either `true` or `false`, instead of just leaving the property out if they don't want percentage colours. Is that really what we want?

Reply:
I think it should be optional :slightly_smiling_face:

Reply:
It seems it has already been made optional in a later fixup 👍